packages/driver/test/cypress/integration/dom/visibility_spec.ts

```diff
@@ -835,6 +835,11 @@ describe('src/cypress/dom/visibility', () => {
           expect(el).to.be.visible
         })
 
+        it('is visible even if there is a dangling element in the tree', () => {
+          cy.visit('/fixtures/dangling-element.html')
+          cy.get('.hello')
+        })
+
```

Comment:
```suggestion
		// https://github.com/cypress-io/cypress/issues/6745
        it('is visible even if there is a dangling element in the tree', () => {
          cy.visit('/fixtures/dangling-element.html')
          cy.get('.hello')
        })

```

Reply:
whenever you're writing a test that fixes a specific issue always note it as a comment so other team members understand its connection to the underlying issue

Reply:
Done.